src/helpers/parseRefType.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { parseRefType } from './parseRefType';
+
+const scenarios: [string, { input: string[]; output: string }][] = [
+    ['empty array', { input: [], output: (undefined as unknown) as string }],
+    ['array with one entry', { input: ['first'], output: `first` }],
+    ['array with more than one entries', { input: ['first', 'second'], output: `second` }],
+];
+
+describe(`parseRefType`, () => {
+    it.each(scenarios)(`should return expected string when for %s`, (_, { input, output }) => {
+        const result = parseRefType(input);
+
+        expect(result).toBe(output);
+    });
+});
```

Comment:
1. What I meant by `it.each` is this:
```ts
import { parseRefType } from './parseRefType';

describe(`parseRefType`, () => {
    it.each([
        ['empty array', { input: [], output: undefined }],
        ['array with one entry', { input: ['first'], output: `first` }],
        ['array with more than one entries', { input: ['first', 'second'], output: `second` }],
    ])(`should return expected string when for %s`, (_, { input, output }) => {
        const result = parseRefType(input);

        expect(result).toBe(output);
    });
});
```
in this case you will not need to describe  simple types.
2. Please check your`describe` test names, you got 3 same `describe` in 3 diff files:
<img width="601" alt="image" src="https://github.com/LandrAudio/openapi-codegen-typescript/assets/22501553/90b23993-66d8-4464-b756-e3055fe4aea2">

3. Do you really need only one `describe` per file?

`describe` is needed for common suites, like
```ts
describe(`some 1`, () => {
it...
it...
it...

});

describe(`some 2`, () => {
it...
it...
it...

})

describe(`some 3`, () => {
it...
it...
it...

})

// But most of the time you are using only one
describe(`i think it is redundant`, () => {
it...
it...
it...
})

```

<img width="654" alt="image" src="https://github.com/LandrAudio/openapi-codegen-typescript/assets/22501553/1f868c51-940d-41f7-8442-7596cbe90aff">

Reply:
personal preference to wrap my test suite with a describe, I don't mind remove them

Reply:
for point 1, also personal preference, but I found it more 'clean' to have all audits store in a `const scenarios`. 
In the example you give  it's not too bad because the audit config are on one line. But often we need more than one line per audit and I wanted to keep things consistent between the files.